app/dojo.py

```diff
@@ -0,0 +1,83 @@
+from os import sys, path
+sys.path.append(path.dirname(path.dirname(
+	path.dirname(path.abspath(__file__)))))
+from app.office import Office
+from app.livingspace import LivingSpace
+from app.room import Room
+from app.utilities import Utilities
+from app.staff import Staff
+from app.fellow import Fellow
+from app.file import File
+from app.person import Person
+
+
+class Dojo(object):
+	"""docstring for Dojo"""
+
+	def __init__(self):
+		pass
+
+	def create_room(room_type, room_names):
+		type_of_room = room_type.strip().lower()
+		if type_of_room == "office":
+			return Office.create_office(room_names)
+		elif type_of_room == "livingspace":
+			return LivingSpace.create_livingspace(room_names)
+		else:
+			print("Invalid type of room")
+
+	def allocate_room(person, wants_accomodation=None):
+		try:
+			if((wants_accomodation is None) or (wants_accomodation.lower() != "y")):
+				print(Office.allocate_office(person))
+			else:
+				print("{}\n{}".format(Office.allocate_office(person),
+									  LivingSpace.allocate_livingspace(person)))
+		except Exception as e:
```

Comment:
Broad except is usually frowned upon.. You need to specify the exact exception you are trying to catch.

e.g 

```python
try:
    foo = 5/0
except ZeroDivisionError:
    print('Sorry, but you cant divide by zero')
```